docs/src/components/Guidelines/GuidelinesSideBySide.tsx

```diff
@@ -24,6 +24,10 @@ const StyledContainer = styled.div`
     background-color: ${theme.orbit.paletteCloudLight};
     border-radius: 16px;
     width: 100%;
+
+    p + p {
+      margin-top: ${theme.orbit.spaceSmall};
+    }
```

Comment:
I can add a comment. This was a problem  I think we discussed with Guidelines not inheriting properties from StyledProse. Instead of fixing it there, you suggested adding something here. The comment should help us remember.